src/GNDStk/Node/test/add.test.cpp

```diff
@@ -79,7 +79,12 @@ SCENARIO("Testing GNDStk Node add()") {
       n.add(dbl);
       n.add(dbl,1.23);
       n.add(dim);
+      // these need the = TYPE on class T in the metadata node.add() functions:
       n.add(dim,{321,987});
+      auto optdim = keyword.meta<std::optional<dimensions2d>>("OptDim");
+      auto defdim = defaulted<dimensions2d>({1,2})/keyword.meta<>("DefDim");
+      n.add(optdim,{321,987});
+      n.add(defdim,{321,987});
```

Comment:
I'm not sure what you are testing here. You `add` some things, but you don't verify that anything was added. In short, you are just verifying that something compiles. Am I misunderstanding what you are doing here?

Reply:
In this case, actually, I *am* just ensuring that it compiles. For a brace-enclosed initializer to automatically become a `std::vector`, in this context, requires some extra template chicanery. But I can certainly go in and also, as a secondary goal, make sure that the values were added.